internal/kafka/internal/services/data_plane_kafka.go

```diff
@@ -77,6 +77,7 @@ func (d *dataPlaneKafkaService) UpdateDataPlaneKafkaService(ctx context.Context,
 			e = d.persistKafkaRoutes(kafka, ks, cluster)
 			if e == nil {
 				e = d.setKafkaClusterReady(kafka)
+				kafka.AdminApiServerURL = ks.AdminServerURI
```

Comment:
Hey @machi1990, could you also check this out?

Reply:
We need to issue the database upgrade so I feel like this might not be enough to update the value.
Something like what's done in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/40c368eb97e25b21876874fa0ac4a090320469b5/internal/kafka/internal/services/data_plane_kafka.go#L125

It quite possible that we can do everything in one go: by moving what you current have in line 80 before line 79. 
That will mean that the `kafka` has the admin api server url assigned. 
And then in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/40c368eb97e25b21876874fa0ac4a090320469b5/internal/kafka/internal/services/data_plane_kafka.go#L125 we could also pass in the value of the admin api server re-using the value assigned in line 80.
```go
	err = d.kafkaService.Updates(kafka, map[string]interface{}{"admin_api_server_url": kafka.AdminApiServerURL, "failed_reason": "", "status": constants2.KafkaRequestStatusReady.String()})

```

Reply:
Thanks for the explanations and assistance with this @machi1990.

Reply:
Sorry for the delay in getting back. 

> Or will a single call suffice?

Let's use one call to the database. What do you think? @VanillaSpoon

Reply:
Perfect, I think that's reasonable alright. Thanks.

Reply:
Awesome